mb-solr/src/main/java/org/musicbrainz/search/solrwriter/MBError.java

```diff
@@ -1,8 +1,8 @@
 package org.musicbrainz.search.solrwriter;
 
-import jakarta.xml.bind.annotation.XmlElement;
-import jakarta.xml.bind.annotation.XmlRootElement;
-import jakarta.xml.bind.annotation.XmlType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
```

Comment:
Because I switched back to version 2.3.0 of jaxb-impl, we now stay with the `javax.xml.bind` package.